SpringProject/src/main/java/com/gradproject2019/userConference/persistance/UserConferenceId.java

```diff
@@ -0,0 +1,36 @@
+package com.gradproject2019.userConference.persistance;
+
+
+import java.io.Serializable;
+
+
+public class UserConferenceId implements Serializable {
```

Comment:
What's the difference between this and UserConference?

Reply:
I believe this file is required because we are using a composite primary key, so this defines what is contained in the key. Before we were not using this and the project would not compile so it does appear to make a difference.

Reply:
Would you mind explaining this to me in person?